### PR TITLE
Auth section

### PR DESCRIPTION
**Pull request by muralibasani**
Adding two sections.
- Authentication
- Kafka cluster migration
- Tenants
- Analytics
- Settings
- Others

### CODE REVIEW

File: docs/docs/howto/authentication/azure-ad.rst [hunk 1]
```diff
@@ -0,0 +1,56 @@
+Azure AD
+========
+
+Users can login with their credentials configured in Azure Active directory. On the login screen user, is prompted with a login option for Azure AD.
+
+After clicking on the login button, user is provided with an Azure pop up window with options to provide his/her credentials or select an Azure account.
+
+* Configuration
+
+1. Make sure Klaw is running in secure mode::
```
Comment (aindriu-aiven) [inline docs/docs/howto/authentication/azure-ad.rst:10]: I think mentioning that this is all configured in the application.properties of klaw-core would help. 
It is mentioned in point 4 but i think moving it up would be great.

File: docs/docs/howto/authentication/google-account.rst [hunk 2]
```diff
@@ -0,0 +1,54 @@
+Google Account Login
+====================
+
+Users can login with their Google account credentials. On the login screen user, is prompted with a login option for Google account.
+
+After clicking on the login button, user is provided with an Google account pop up window with options to provide his/her credentials or select a Google account.
+
+* Configuration
+
```
Comment (aindriu-aiven) [inline docs/docs/howto/authentication/google-account.rst:9]: Same comment here & also for third party etc entries below.

Comment (muralibasani) [reply docs/docs/howto/authentication/google-account.rst:9]: @aindriu-aiven pushed new changes

**Review by aindriu-aiven — APPROVED** [review]

**Review by aindriu-aiven — APPROVED** [review]
This looks great

**Pull request merged**